src/gt4sd/algorithms/generation/hugging_face/implementation.py

```diff
@@ -137,6 +139,7 @@ def adjust_length_to_model(length: int, maximum_sequence_length: int):
     "xlnet": (XLNetLMHeadModel, XLNetTokenizer, prepare_prefix_input),
     "transfo-xl": (TransfoXLLMHeadModel, TransfoXLTokenizer, prepare_prefix_input),
     "xlm": (XLMWithLMHeadModel, XLMTokenizer, None),
+    "t5": (AutoModelForSeq2SeqLM, AutoTokenizer, None),
```

Comment:
should we consider using another key name as it seems more generic than t5, maybe auto-seq2seq-lm?